patchkit-launcher-qt-src/src/localpatcherdata.cpp

```diff
@@ -136,7 +136,7 @@ void LocalPatcherData::install(const QString& t_downloadedPath, const Data& t_da
     IOUtils::writeTextToFile(locations.patcherIdInfoFilePath(), getPatcherId(t_data));
 }
 
-void LocalPatcherData::start(const Data& t_data)
+void LocalPatcherData::start(const Data& t_data, bool isOnline)
```

Comment:
Another proof that enum would be better - mostly you are using `isOffline` and here we can see `isOnline` which seems not consistent.